java-bootcamp-bahia-t1/src/MailListSubject.java

```diff
@@ -0,0 +1,9 @@
+public interface MailListSubject {

```

Comment:
Please could you write two lines explaining which pattern did you use to modelate item 6? and Why did you choose it?